### PR TITLE
Enhance KenyEMR to have a Lab Order for SARI/ILI Samples

### PR DESCRIPTION
**Pull request by gabriel090**
nasai1
nasal
nasal2

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/kenyaemrorderentry/fragment/controller/patientdashboard/CreateLabsOrdersFragmentController.java [hunk 1]
```diff
@@ -143,13 +143,20 @@ public void labOrdersConceptPanels(@SpringBean("conceptService") ConceptService
                 // conceptService.getConcept(159645)
 
 
+        );
+        // Build panels for Mucus
+        List<Concept> mucusTestPanels = Arrays.asList(
+//                conceptService.getConcept(163363),
+//                conceptService.getConcept(162614),
+//                conceptService.getConcept(163364)
+
```
Comment (makombe) [inline omod/src/main/java/org/openmrs/module/kenyaemrorderentry/fragment/controller/patientdashboard/CreateLabsOrdersFragmentController.java:152]: Remove the commented out lines

File: omod/src/main/java/org/openmrs/module/kenyaemrorderentry/page/controller/LabOrdersPageController.java [hunk 2]
```diff
@@ -186,7 +194,24 @@ public void labOrdersConceptPanels(@SpringBean("conceptService") ConceptService
...
+                        concService.getConcept(166556), // Flu Antigen Test
+                        //   concService.getConcept(1305), // hiv viral load Qualitative
+                        concService.getConcept(165895)// cd4 counts
+                ));
+        JSONArray nasopharyngealSWABMonitoring = buildTestPanelWithoutPanelConcept("Mucus", labTestJsonPayload,
+                "Nasopharyngeal Swab", Arrays.asList(
+                        concService.getConcept(166556), // Flu Antigen Test
+                        //   concService.getConcept(1305), // hiv viral load Qualitative
+                        concService.getConcept(165895)// cd4 counts
+                ));
```
Comment (makombe) [inline omod/src/main/java/org/openmrs/module/kenyaemrorderentry/page/controller/LabOrdersPageController.java:214]: Update the comments, I think the concepts are not viral load or cd4 counts related

**Review by makombe — CHANGES_REQUESTED** [review]
@gabriel090  I think also you will need to provide custom answers for for results for `Cerebrospinal fluid Cryptococcus antigen assay` test or else there will be no way the results will be captured or entered. The drop down will be blank.

**Comment by gabriel090** [comment]
Ok

File: omod/src/main/java/org/openmrs/module/kenyaemrorderentry/fragment/controller/patientdashboard/CreateLabsOrdersFragmentController.java [hunk 3]
```diff
@@ -205,6 +212,13 @@ public void labOrdersConceptPanels(@SpringBean("conceptService") ConceptService
                         concService.getConcept(163722), // hiv rapid test
                         concService.getConcept(844) // DNA PCR
                 ));
+        JSONArray nasalSwabMonitoring = buildTestPanelWithoutPanelConcept("Mucus", labTestJsonPayload,
+                " NASAL SWAB", Arrays.asList(
+                        concService.getConcept(166556), // hiv viral load
+                        //   concService.getConcept(1305), // hiv viral load Qualitative
```
[the remark this answers is not in the document]
Comment (gabriel090) [reply omod/src/main/java/org/openmrs/module/kenyaemrorderentry/fragment/controller/patientdashboard/CreateLabsOrdersFragmentController.java:218]: Done Rectified

[the remark this answers is not in the document]
Comment (gabriel090) [reply omod/src/main/java/org/openmrs/module/kenyaemrorderentry/fragment/controller/patientdashboard/CreateLabsOrdersFragmentController.java:218]: Waiting for Cosmas to give to rectify this Cerebrospinal fluid Cryptococcus antigen assay

**Comment by makombe** [comment]
@gabriel090  any updates on this ticket? are you still working on it?

**Comment by gabriel090** [comment]
It's still on hold

**Pull request closed without merging**